.changeset/olive-maps-join.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': minor
```

Comment:
this should have been `patch`

Reply:
thanks for the info. :) btw, this is not clear reading from the readme: https://github.com/sveltejs/kit#changelogs

Reply:
Added!